### ISSUE TITLE
Deprecated code in nova_wiki.php causing an error

### ISSUE THREAD

**Issue by c0dezer019** [anodyne/nova#303]
Editing a page on the wiki generates this error (so far the only place that it pops up).
```
A PHP Error was encountered
Severity: 8192

Message: explode(): Passing null to parameter #2 ($string) of type string is deprecated

Filename: controllers/nova_wiki.php

Line Number: 948

Backtrace:

File: /home/proje184/ares.projectrazer.org/nova/modules/core/controllers/nova_wiki.php
Line: 948
Function: explode

File: /home/proje184/ares.projectrazer.org/index.php
Line: 330
Function: require_once
```

**Comment by c0dezer019** [comment]
I fixed the issue. I don't know if it breaks anything else, but that error went away.

I replaced line 948 with $cats = (! empty($p->draft_categories)) ? explode(',', $p->draft_categories) : '';

I will clone the repo and implement the fix into a PR, if that's alright.

**Comment by agentphoenix** [comment]
@c0dezer019 Sorry I missed this! Definitely ping me directly or find me on Discord if you don't hear back from me on issues or PRs.

That code is pretty close. I've added a fix in 2.7.10 that should address the issue, but if you still see problems, let me know.
